app/controllers/users_controller.rb

```diff
@@ -0,0 +1,49 @@
+class UsersController < ApplicationController
+  def index
+    users = User.all
+
+    render json: users, status: 200
+  end
+
+  def show
+    user = User.find(params[:id])
+
+    render json: user, status: 200
+  end
+
+  def create
+    user = User.new(user_params)
+
+    if user.save!
```

Comment:
You expect [save!](http://devdocs.io/rails~5.0/activerecord/persistence#method-i-save-21) to return `false` if it fails, but it will throw an exception.  You probably meant [save](http://devdocs.io/rails~5.0/activerecord/persistence#method-i-save)